patches/tModLoader/Terraria/NPC.cs.patch

```diff
@@ -51,8 +51,84 @@
 +	/// Has nothing to do with the speed that this NPC travels, that is dictated by AI code adjusting <see cref="Entity.velocity"/>.
 +	/// </summary>
  	public float stepSpeed;
- 	private static float gravity = 0.3f;
++
++	/// <summary>
++	/// Used to keep vanilla and modded gravity effects working neatly
++	/// </summary>
++	private float internalGravityBase = 0.3f;
++
++	/// <summary>
++	/// Modify this value in order to change the NPCs active gravity, this can be done in AI as gravity values are reset slightly beforehand, and used slightly after.
++	/// </summary>
++	public MultipliableFloat Gravity = MultipliableFloat.One;//TML: #AccessibleNPCGravity, added
```

Comment:
Having both `Gravity` and `gravity` could be confusing. Suggestions:
- Call this `GravityMultiplier`
or
- make `gravity` an internal wrapper around `Gravity.Value`